src/main/java/io/servertap/api/v1/ServerApi.java

```diff
@@ -374,4 +307,62 @@ public static void listPlugins(Context ctx) {
 
         ctx.json(pluginList);
     }
+
+    @OpenApi(path = "/v1/server/ops", method = HttpMethod.POST, summary = "Sets a specific player to Op", tags = {
+            "Player" }, formParams = {
+                    @OpenApiFormParam(name = "playerUuid"), }, responses = { @OpenApiResponse(status = "200") })
+    public static void opPlayer(Context ctx) {
+        if (ctx.formParam("playerUuid").isEmpty()) {
+            throw new InternalServerErrorResponse(Constants.PLAYER_MISSING_PARAMS);
+        }
+        org.bukkit.OfflinePlayer player = Bukkit.getOfflinePlayer(UUID.fromString(ctx.formParam("playerUuid")));
+        if (player == null) {
+            throw new InternalServerErrorResponse(Constants.PLAYER_NOT_FOUND);
+        }
+        player.setOp(true);
+        ctx.json("success");
+    }
+
+    @OpenApi(path = "/v1/server/ops", method = HttpMethod.DELETE, summary = "Removes Op from a specific player", tags = {
+            "Player" }, formParams = {
+                    @OpenApiFormParam(name = "playerUuid"), }, responses = { @OpenApiResponse(status = "200") })
+    public static void deopPlayer(Context ctx) {
+        if (ctx.formParam("playerUuid").isEmpty()) {
+            throw new InternalServerErrorResponse(Constants.PLAYER_MISSING_PARAMS);
+        }
+        org.bukkit.OfflinePlayer player = Bukkit.getOfflinePlayer(UUID.fromString(ctx.formParam("playerUuid")));
+        if (player == null) {
+            throw new InternalServerErrorResponse(Constants.PLAYER_NOT_FOUND);
+        }
+        player.setOp(false);
+        ctx.json("success");
+    }
+
+    @OpenApi(path = "/v1/players/op", method = HttpMethod.GET, summary = "Get all op players", tags = {
+            "Player" }, responses = {
+                    @OpenApiResponse(status = "200", content = @OpenApiContent(from = io.servertap.api.v1.models.OfflinePlayer.class, isArray = true)) })
+    public static void getOps(Context ctx) {
+        org.bukkit.OfflinePlayer players[] = Bukkit.getOfflinePlayers();
+        ArrayList<io.servertap.api.v1.models.OfflinePlayer> opedPlayers = new ArrayList<io.servertap.api.v1.models.OfflinePlayer>();
+        for (org.bukkit.OfflinePlayer player : players) {
+            if (!player.isOp()) {
+                return;
```

Comment:
This should be `continue` instead of `return`. `continue` will just go to the next iteration of the for loop, whereas `return` will exit right out of `getOps` meaning that any players that hadn't been looked at yet wouldn't get processed.

Reply:
My bad, I'll fix that